components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/AuthenticationInvocationResponseClassProvider.java

```diff
@@ -0,0 +1,24 @@
+package org.wso2.carbon.identity.application.authenticator.adapter;
+
+import org.wso2.carbon.identity.action.execution.ActionInvocationResponseClassProvider;
+import org.wso2.carbon.identity.action.execution.model.ActionType;
+import org.wso2.carbon.identity.action.execution.model.ResponseData;
+import org.wso2.carbon.identity.application.authenticator.adapter.model.AuthenticatedUserData;
+
+/**
+ * This class provide classes for the action invocation responses defined for the AUTHENTICATION action type.
```

Comment:
Let's mention that this provide the class extending ResponseData of ActionInvocationSuccessResponse to deserialize the response payload from the external service